docs/usage/configuration-options.md

```diff
@@ -1556,6 +1556,23 @@ To adjust it down to 10s for all queries, do this:
 }
 ```
 
+### httpsCertificateAuthority
+
+`httpsCertificateAuthority` overrides the trusted CA certificate that defaults to the curated list of well-known CAs by Mozilla.
+
+### httpsPrivateKey
+
+Specifies the private key in PEM format for mTLS authentication.
+
+<!-- prettier-ignore -->
+!!! warning
+    Do _not_ specify your private key directly here, to avoid losing confidentiality completely.
+    Use [secrets](https://docs.renovatebot.com/self-hosted-configuration/#secrets) to pass it down securely instead.
+
+### httpsCertificate
+
+Specifies the certificate chain in PEM format for mTLS authentication.
```

Comment:
I don't know what the words _certificate chain_, _PEM format_ or mTLS means, just by reading the text.

Is it safe to assume the intended audience knows these words and concepts? Or should we link to some background information?

I'll put the abbreviations into the Renovate docs glossary file once this PR is merged. That means the docs will have an explanation of the abbreviation when you hover over it with the mouse.

Reply:
background information is available in [`got` ](https://github.com/sindresorhus/got/blob/main/documentation/5-https.md#certificate)

Reply:
I've added the links from the got page above